examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/ConfigureFragment.kt

```diff
@@ -56,6 +58,26 @@ class ConfigureFragment : Fragment() {
             }.collect()
         }
 
+        @Suppress("EmptyFunctionBlock")
+        binding.proxyUrlInput.addTextChangedListener(object : TextWatcher {
```

Comment:
I added this so we could configure the proxy setting from before the SDK was configured. This means we will update the property on every change, which is probably fine.

Reply:
it's fine for a sample app. Would listening to `onFocusChange` also do the trick, but without the intermediate calls while editing?

Reply:
There are some situations where the edittext doesn't lose focus, causing the proxy url to not be updated in time. For example, if you are editting the proxy url edittext and then click a button, the focus isn't lost. Updating on text change was the quickest alternative I could find, even though it's clearly not optimal